apps/journeys-admin/src/components/JourneyVisitorsList/FilterDrawer/ExportDialog/ExportDialog.spec.tsx

```diff
@@ -0,0 +1,435 @@
+    user = userEvent.setup()
+    ;(useJourneyEventsExport as jest.Mock).mockReturnValue({
+      exportJourneyEvents: mockExportJourneyEvents
+    })
+  })
+
+  it('should render correctly with initial state', async () => {
+    render(
+      <MockedProvider mocks={[mockJourneyCreatedAt]}>
+        <SnackbarProvider>
+          <ExportDialog {...defaultProps} />
+        </SnackbarProvider>
+      </MockedProvider>
+    )
+
+    await waitFor(() => {
+      expect(
+        screen.getByDisplayValue(journeyCreatedAt.split('T')[0])
+      ).toBeInTheDocument()
+    })
+    expect(screen.getByLabelText('All')).toBeChecked()
+    expect(screen.getByLabelText('Journey Start')).toBeChecked()
+    expect(screen.getByRole('button', { name: 'Export (CSV)' })).toBeEnabled()
+  })
+
+  it('should call export function with default filters on button click', async () => {
+    render(
+      <MockedProvider mocks={[mockJourneyCreatedAt]}>
+        <SnackbarProvider>
+          <ExportDialog {...defaultProps} />
+        </SnackbarProvider>
+      </MockedProvider>
+    )
+
+    await waitFor(() => {
+      expect(
+        screen.getByDisplayValue(journeyCreatedAt.split('T')[0])
+      ).toBeInTheDocument()
+    })
+
+    const exportButton = screen.getByRole('button', { name: 'Export (CSV)' })
+    await user.click(exportButton)
+
+    await waitFor(() => {
+      expect(mockExportJourneyEvents).toHaveBeenCalledWith({
+        journeyId: 'journey1',
+        filter: {
+          typenames: [
+            EventType.JourneyViewEvent,
+            EventType.ChatOpenEvent,
+            EventType.TextResponseSubmissionEvent,
+            EventType.RadioQuestionSubmissionEvent,
+            EventType.ButtonClickEvent,
+            EventType.SignUpSubmissionEvent,
+            EventType.VideoStartEvent,
+            EventType.VideoPlayEvent,
+            EventType.VideoPauseEvent,
+            EventType.VideoCompleteEvent,
+            EventType.VideoProgressEvent
+          ],
+          periodRangeStart: journeyCreatedAt,
+          periodRangeEnd: expect.any(String) // End date defaults to now
+        }
+      })
+    })
+    expect(mockOnClose).toHaveBeenCalled()
+  })
+
+  it('should call export function with updated filters after changing selections', async () => {
+    render(
+      <MockedProvider mocks={[mockJourneyCreatedAt]}>
+        <SnackbarProvider>
+          <ExportDialog {...defaultProps} />
+        </SnackbarProvider>
+      </MockedProvider>
+    )
+
+    await waitFor(() => {
+      expect(
+        screen.getByDisplayValue(journeyCreatedAt.split('T')[0])
+      ).toBeInTheDocument()
+    })
+
+    await user.click(screen.getByTestId('video-events-expander'))
+    await user.click(screen.getByLabelText('Chat Open'))
+    await user.click(screen.getByLabelText('Start'))
+
+    await waitFor(() => {
+      expect(screen.getByLabelText('Start')).not.toBeChecked()
+      expect(screen.getByLabelText('Chat Open')).not.toBeChecked()
+    })
```

Comment:
Only 1 expect per `waitFor`
```suggestion
    await waitFor(() => {
      expect(screen.getByLabelText('Start')).not.toBeChecked()
    })
      expect(screen.getByLabelText('Chat Open')).not.toBeChecked()
```